src/shared-components/progressBar/__snapshots__/test.tsx.snap

```diff
@@ -0,0 +1,137 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<ProgressBar /> UI snapshot renders with custom props 1`] = `
+@keyframes animation-0 {
+  from {
+    -webkit-transform: translateX(-100%);
+    -ms-transform: translateX(-100%);
+    transform: translateX(-100%);
+  }
+
+  to {
+    -webkit-transform: translateX(-5%);
+    -ms-transform: translateX(-5%);
+    transform: translateX(-5%);
+  }
+}
+
+@keyframes animation-0 {
```

Comment:
Do we know why this is showing two separate keyframes with the same name? Is it fixing the typo? Are we duplicating the style somehow (maybe because there was a typo, we didn't see it working, and so activated it some other way?)

Reply:
the typo influenced the `transition` css not the animation.  I analyzed and see no apparent issues. Remeber we updated Jest to a new mayor version so that could be it also the default props test looks the same (except for the defaultTiming fix) and only defines it once I think is just a snapshot creation thing